Intake/Onboarding/InterestingModules.swift

```diff
@@ -16,24 +16,21 @@ struct InterestingModules: View {
     
     var body: some View {
         SequentialOnboardingView(
-            title: "INTERESTING_MODULES_TITLE",
-            subtitle: "INTERESTING_MODULES_SUBTITLE",
+            title: "The Intake Process",
+            subtitle: "Intake works in 3 simple steps...",
             content: [
                 SequentialOnboardingView.Content(
-                    title: "INTERESTING_MODULES_AREA1_TITLE",
-                    description: "INTERESTING_MODULES_AREA1_DESCRIPTION"
+                    title: "Log in to Firebase",
```

Comment:
Please leave out the technical details like "Firebase", the onboarding flow should be as seamless as possible.